src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -13,75 +13,84 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
-<%@ page import="java.util.List" %>
-<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="java.util.List"%>
+<%@ page import="codeu.model.data.Conversation"%>
 
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Conversations</title>
-  <link rel="stylesheet" href="/css/main.css">
+<title>Conversations</title>
+<link rel="stylesheet" href="/css/main.css">
```

Comment:
Lines 22, 23 were indented originally because they are within the \< head \> block.